semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -244,10 +244,55 @@ def __init__(self, group):
         self.members = UnitMember.parse(group.get("members"))
 
 
-class MetricSemanticConvention(BaseSemanticConvention):
+class MetricGroupSemanticConvention(BaseSemanticConvention):
+    GROUP_TYPE_NAME = "metric_group"
+
+
+class MetricSemanticConvention(MetricGroupSemanticConvention):
     GROUP_TYPE_NAME = "metric"
 
-    allowed_keys = ()
+    allowed_keys: Tuple[str, ...] = BaseSemanticConvention.allowed_keys + (
+        "metric_name",
+        "unit",
+        "instrument",
+    )
+
+    yaml_to_markdown_instrument_repr: Dict[str, str] = {
+        "counter": "Counter",
+        "updowncounter": "UpDownCounter",
+        "histogram": "Histogram",
+        "gauge": "Gauge",
+    }
+
+    allowed_instruments: Tuple[str, ...] = tuple(
+        yaml_to_markdown_instrument_repr.keys()
+    )
+
+    def __init__(self, group):
+        super().__init__(group)
+        self.metric_name = group.get("metric_name")
+        self.unit = group.get("unit")
+        self.instrument = group.get("instrument")
+        self.instrument_markdown_fmt = self.yaml_to_markdown_instrument_repr.get(
```

Comment:
Maybe do this after `validate()` and use `[]` instead of `.get`

Reply:
I've removed this line, see https://github.com/open-telemetry/build-tools/pull/79#discussion_r1035414725